Toolbox unit tests are broken

```
======================================================================
FAIL: test_remove_object_metadata_row (tests.spine_db_editor.mvcmodels.test_item_metadata_table_model.TestItemMetadataTableModelWithExistingData)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Spine-Toolbox/Spine-Toolbox/tests/spine_db_editor/mvcmodels/test_item_metadata_table_model.py", line 280, in test_remove_object_metadata_row
    self.assertEqual(
AssertionError: {<spi[70 chars]: [{'id': 1, 'entity_id': 1, 'metadata_id': 1,[109 chars] 2}]} != {<spi[70 chars]: [{'entity_id': 1, 'id': 1, 'metadata_id': 1,[13 chars] 2}]}
Diff is 1231 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_remove_object_parameter_value_metadata_row (tests.spine_db_editor.mvcmodels.test_item_metadata_table_model.TestItemMetadataTableModelWithExistingData)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Spine-Toolbox/Spine-Toolbox/tests/spine_db_editor/mvcmodels/test_item_metadata_table_model.py", line 293, in test_remove_object_parameter_value_metadata_row
    self.assertEqual(
AssertionError: {<spi[70 chars]: [{'id': 1, 'parameter_value_id': 1, 'metadat[190 chars] 2}]} != {<spi[70 chars]: [{'parameter_value_id': 1, 'id': 1, 'metadat[22 chars] 2}]}
Diff is 1576 characters long. Set self.maxDiff to None to see it.
````

--- a/tests/spine_db_editor/mvcmodels/test_item_metadata_table_model.py
+++ b/tests/spine_db_editor/mvcmodels/test_item_metadata_table_model.py
@@ -278,7 +278,20 @@ class TestItemMetadataTableModelWithExistingData(unittest.TestCase):
             self._model.removeRows(0, 1)
             waiter.wait()
         self.assertEqual(
-            change_listener.removed_items, {self._db_map: [{"entity_id": 1, "id": 1, "metadata_id": 1, "commit_id": 2}]}
+            change_listener.removed_items,
+            {
+                self._db_map: [
+                    {
+                        "entity_id": 1,
+                        "id": 1,
+                        "metadata_id": 1,
+                        "metadata_name": "source",
+                        "commit_id": 2,
+                        "entity_name": "my_object",
+                        "metadata_value": "Fountain of objects",
+                    }
+                ]
+            },
         )
         self._model.remove_item_metadata(change_listener.removed_items)
         self.assertEqual(self._model.rowCount(), 1)
@@ -292,7 +305,21 @@ class TestItemMetadataTableModelWithExistingData(unittest.TestCase):
             waiter.wait()
         self.assertEqual(
             change_listener.removed_items,
-            {self._db_map: [{"parameter_value_id": 1, "id": 1, "metadata_id": 3, "commit_id": 2}]},
+            {
+                self._db_map: [
+                    {
+                        "parameter_value_id": 1,
+                        "id": 1,
+                        "metadata_id": 3,
+                        "commit_id": 2,
+                        "entity_name": "my_object",
+                        "metadata_name": "source",
+                        "metadata_value": "Fountain of object values",
+                        "parameter_name": "object_parameter",
+                        "alternative_name": "Base",
+                    }
+                ]
+            },
         )
         self._model.remove_item_metadata(change_listener.removed_items)
         self.assertEqual(self._model.rowCount(), 1)

